 * Serving Flask app 'app'
 * Debug mode: on
/home/gaurav/secureguard/secureguard/app.py:64: DeprecationWarning: datetime.datetime.utcnow() is deprecated and scheduled for removal in a future version. Use timezone-aware objects to represent datetimes in UTC: datetime.datetime.now(datetime.UTC).
  ts = datetime.utcnow().isoformat()
/home/gaurav/secureguard/secureguard/app.py:87: DeprecationWarning: datetime.datetime.utcnow() is deprecated and scheduled for removal in a future version. Use timezone-aware objects to represent datetimes in UTC: datetime.datetime.now(datetime.UTC).
  cutoff = (datetime.utcnow() - timedelta(seconds=60)).isoformat()
 * Debugger is active!
 * Debugger PIN: 139-863-190
127.0.0.1 - - [05/Sep/2025 13:01:50] "[35m[1mGET / HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/home/gaurav/secureguard/secureguard/venv/lib/python3.13/site-packages/flask/app.py", line 1536, in __call__
    return self.wsgi_app(environ, start_response)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gaurav/secureguard/secureguard/venv/lib/python3.13/site-packages/flask/app.py", line 1514, in wsgi_app
    response = self.handle_exception(e)
  File "/home/gaurav/secureguard/secureguard/venv/lib/python3.13/site-packages/flask/app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/gaurav/secureguard/secureguard/venv/lib/python3.13/site-packages/flask/app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/gaurav/secureguard/secureguard/venv/lib/python3.13/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/gaurav/secureguard/secureguard/venv/lib/python3.13/site-packages/flask/app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/home/gaurav/secureguard/secureguard/app.py", line 90, in index
    events = conn.execute("SELECT timestamp, message FROM events ORDER BY id DESC LIMIT 15").fetchall()
             ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
sqlite3.OperationalError: no such column: timestamp
127.0.0.1 - - [05/Sep/2025 13:01:50] "GET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
127.0.0.1 - - [05/Sep/2025 13:01:50] "GET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
127.0.0.1 - - [05/Sep/2025 13:01:50] "GET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
127.0.0.1 - - [05/Sep/2025 13:01:50] "GET /?__debugger__=yes&cmd=resource&f=console.png&s=LiEOT4ucN1FTk5wI2Jkl HTTP/1.1" 200 -
